Mathlib/Algebra/Module/Equiv.lean

```diff
@@ -342,11 +342,11 @@ def trans
 #align linear_equiv.trans LinearEquiv.trans
 
 /-- The notation `e₁ ≪≫ₗ e₂` denotes the composition of the linear equivalences `e₁` and `e₂`. -/
-infixl:80 " ≪≫ₗ " =>
+notation3:80 (name := transNotation) e₁:81 " ≪≫ₗ " e₂:80 =>
```

Comment:
I always forget the exact precedences to use to implement left- vs right-associativity. I just read the `infixl` elaborator code as a refresher, and I *think* you did the `infixr` encoding. Is that intentional?

Reply:
I don't mind if you decide `infixr` is the right way to go, but if so let's be sure to document the change in the PR description.

I'm guessing you do want `infixl` though since equivalences have reverse-order composition, right?

Reply:
Ah, this was indeed a mistake, based on assuming that `trans` and `comp` used the same convention.

Is this correct?

```suggestion
notation3:80 (name := transNotation) e₁:80 " ≪≫ₗ " e₂:81 =>
```

Reply:
I believe so! It's worth you independently verifying which way it's supposed to be. The macro isn't so bad to read, and the key feature is just which side gets the +1.